pkg/charts/charts_test.go

```diff
@@ -174,6 +174,12 @@ var _ = Describe("Chart package test", func() {
 						"ipinip": map[string]interface{}{
 							"enabled": true,
 						},
+						"bpf": map[string]interface{}{
+							"enabled": false,
```

Comment:
Do you want to add any test with the bpf dataplane and/or the iptables cleanup being enabled?

Reply:
Good point, added a test for the ebpf dataplane.